orchestrator-salt/src/main/resources/salt/salt/disks/service/scripts/mount-instance-storage.j2

```diff
@@ -235,6 +240,64 @@ find_format_and_mount_ephemeral_storage_aws() {
     return $((return_code))
 }
 
+find_format_and_mount_ephemeral_storage_gcp() {
```

Comment:
This seems to be a complete copy of [the AWS related bash function](https://github.com/hortonworks/cloudbreak/pull/14989/files#diff-1cc3638f1691bc85d78e3e91944e162beaab56d2a104655bd96bb003b081b16fL181-L236https://github.com/hortonworks/cloudbreak/pull/14989/files#diff-1cc3638f1691bc85d78e3e91944e162beaab56d2a104655bd96bb003b081b16fL181-L236)
Could you please try refactor these two 50liners?